Orbit.h

```diff
@@ -9,45 +9,173 @@
 
 #include <cstring>
 
-// Orbit API (header-only)
+// Orbit Manual Instrumentation API (header-only)
 //
-// While the main feature of Orbit is its ability to dynamically instrument functions, manual
-// instrumentation is still possible using the macros below. These macros call empty functions that
-// Orbit dynamically instruments.
+// While Orbit's main feature is dynamic instrumentation, manual instrumentation can be extremely
+// useful. The macros below allow you to profile sections of functions, track "async" operations,
+// and graph interesting values in Orbit's main capture window.
 //
-// Note: We currently only support strings smaller than 34 characters, longer strings will be
-//       truncated. An upcoming update will allow for arbitrarily long strings.
+// Summary:
+// ORBIT_SCOPE: Profile current scope.
+// ORBIT_START/ORBIT_STOP: Profile sections inside a scope.
+// ORBIT_START_ASYNC/ORBIT_STOP_ASYNC: Profile time spans across scopes or threads.
+// ORBIT_ASYNC_STRING: Provide additional information for an async time span.
+// ORBIT_INT: Graph int values.
+// ORBIT_INT64: Graph int64_t values.
+// ORBIT_UINT: Graph uint32_t values.
+// ORBIT_UINT64: Graph uint64_t values.
+// ORBIT_FLOAT: Graph float values.
+// ORBIT_DOUBLE: Graph double values.
+//
+// Implementation:
+// The manual instrumentation macros call empty "ORBIT_STUB" functions that Orbit dynamically
+// instruments. For manual instrumentation to appear in your Orbit capture, make sure that symbols
+// have been loaded for the manually instrumented modules.
 
 // To disable manual instrumentation macros, define ORBIT_API_ENABLED as 0.
 #define ORBIT_API_ENABLED 1
 
 #if ORBIT_API_ENABLED
 
-// ORBIT_SCOPE: profile current scope.
+// ORBIT_SCOPE: Profile current scope.
+//
+// Overview:
+// ORBIT_SCOPE will profile the time between "now" and the end of the current scope.
+//
+// Note:
+// We limit the maximum number of characters of the "name" parameter to "kMaxEventStringSize". This
+// limitation will be lifted as we roll out a new dynamic instrumentation implementation on Linux.
+//
+// Example Usage: Profile sections of a function:
+//
+// void MyVeryLongFunction() {
+//   DoSomeWork();
+//   if(condition) {
+//     ORBIT_SCOPE("DoSomeMoreWork");
+//     DoSomeMoreWork();
+//   } else {
+//     ORBIT_SCOPE_WITH_COLOR("DoSomeOtherWork", orbit::Color::kLightGreen);
+//     DoSomeOtherWork();
+//   }
+// }
+//
+// Parameters:
+// name: [const char*] Label to be displayed on current time slice (kMaxEventStringSize characters).
+// col: [orbit::Color] User defined color for the current time slice.
```

Comment:
Maybe point out that predefined colors can be found below? 

"(see below for color definitions)"